shell/packages/sandstorm-db/migrations.js

```diff
@@ -312,6 +312,50 @@ function repairEmailIdentityIds() {
   });
 }
 
+function splitAccountUsersAndIdentityUsers() {
+  Meteor.users.find({"identities": {$exists: true}}).forEach(function (user) {
+    if (user.identities.length != 1) {
```

Comment:
If the migration fails half-way through then re-runs, won't this check fail on already-migrated users, since `identities` is gone?